scripts/deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -e
+
+rshell cp ${1} /pyboard/main.py
+rshell 'repl ~ import machine ~ machine.soft_reset()~'
```

Comment:
LGTM, at some point I may add a flag to allow the script to run and attach to the rshell serial output for print debugging purposes.

Basically just running it like this:
```shell
rshell 'repl ~ import main'
```